modules/services/playerctld.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let cfg = config.services.playerctld;
+in {
+  meta.maintainers = maintainers.fendse;
+
+  options.services.playerctld = {
+    enable = mkEnableOption "playerctld daemon";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.playerctl;
+      defaultText = literalExample "pkgs.playerctl";
+      description = "The playerctl package to use";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.user.services.playerctld = {
+      Unit = { Description = "MPRIS media player daemon"; };
+
+      Install = { WantedBy = [ "default.target" ]; };
+
+      Service = {
+        ExecStart = "${cfg.package}/bin/playerctld";
```

Comment:
It does, but mostly just to send commands to a daemon that is already running. The only argument that could be relevant at startup is the ability to ask it to use D-Bus service activation to launch the daemon indirectly instead of just, well, having the process itself be the daemon. I don't think supporting that here is useful, which is why I didn't add it.

Reply:
Thanks for the answer :+1: